pages/docs/manual/latest/migrate-from-bucklescript-reason.mdx

```diff
@@ -16,12 +16,25 @@ ReScript is a rebranding and cleanup of BuckleScript (since `v8.2.0`) & Reason (
 There are lots of exciting improvements in the new syntax (features, speed, error messages, etc.). The upgrade is trivial, backward-compatible and can be done on a per-file basis:
 
 - Upgrade your project to `bs-platform 8.2.0` or later.
-- `node_modules/.bin/bsc -format MyFile.re > MyFile.res`
+- `node_modules/.bin/bsc -format MyFile.re > MyFile.res`.
+- If your new `MyFile.res` looks good, you can delete (or move) `MyFile.re` (otherwise you will have an error `Invalid bsconfig.json implementation and interface have different path names or different cases MyFile vs MyFile`).
 
 **That's it**! `MyFile.re` could be any `.re`, `.rei`, `.ml` and `.mli` file.
 
 Enjoy the improved experience!
 
+### Upgrade an Entire Folder
+
+Please use this with caution. Migrating an entire codebase at once is an uncessary risk.
+It is recommended to migrate small portion of code to ensure correctness.
+That being said, for small codebases or some tiny folders that are safely versionned, you can try the following:
+
+```console
+for f in your-folder/**/*.re; do; node_modules/.bin/bsc -format $f > ${f%.re}.res && rm $f; done;
+```
+
+This command loop on the `.re` files from the `your-folder` folder, convert them with a proper `.res` extension, and ask you if you want to remove the previous file (check the newly created `.res` file first). If you are confident enough, you can add `-f` option to the `rm` command.
```

Comment:
Typos: This command loop**s** ... convert**s** ... and ask**s**

Reply:
Thanks again! Fixed!